app/build.gradle

```diff
@@ -72,6 +75,8 @@ dependencies {
     implementation 'com.squareup.retrofit2:adapter-rxjava:2.1.0'
     implementation 'com.squareup.okhttp3:logging-interceptor:3.9.1'
 
+    implementation 'com.google.firebase:firebase-core:16.0.0'
```

Comment:
Please update to latest (currently not building in Studio 3.3.1)

Reply:
In order to have Crashlytics we will need to have a google-services.json file